test/NuGet.Clients.Tests/NuGet.CommandLine.Test/ConsoleTest.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using NuGet.Test.Utility;
+using Xunit;
+
+namespace NuGet.CommandLine.Test
+{
+    public class ConsoleTest
+    {
+        [Fact]
+        public void TestConsoleWindowWidthNotZero()
+        {
+            Assert.NotEqual(0, new NuGet.Common.Console().WindowWidth);
```

Comment:
This test will only be relevant if someone runs the tests on Linux, with output redirect. I tried messing around with `System.Console.SetOut()` and the like at runtime, but Mono does some [initialization](https://goo.gl/dLCVCa) before that, so it won't apply.